commands/issue.go

```diff
@@ -428,3 +437,55 @@ text is the title and the rest is the description.`, project))
 
 	messageBuilder.Cleanup()
 }
+
+func listLabels(cmd *Command, args *Args) {
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	if args.Noop {
+		ui.Printf("Would request list of labels for %s\n", project)
+		return
+	}
+
+	labels, err := gh.FetchLabels(project)
+	utils.Check(err)
+
+	for _, label := range labels {
+		// TODO add a colorize flag for the labels command
```

Comment:
Would this be enough?

```go
colorize := ui.IsTerminal(os.Stdout)
```

Reply:
I was thinking along the lines of a command line argument to control whether to print in plain text or a colorized string, like `hub issue labels --no-color`. I could add that in plus the `IsTerminal` check?